### PR TITLE
[Hackathon 7th] fix voc1 readme.md in CSMSC 

### PR DESCRIPTION
**Pull request by enkilee**
### PR types

Others
### PR changes

Others
### Describe

明确dataset路径

### CODE REVIEW

File: examples/csmsc/voc1/README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@ This example contains code used to train a [parallel wavegan](http://arxiv.org/a
 ## Dataset
 ### Download and Extract
 Download CSMSC from it's [official website](https://test.data-baker.com/data/index/TNtts/) and extract it to `~/datasets`. Then the dataset is in the directory `~/datasets/BZNSYP`.
+The structure of the folder is listed below.
+```text
+datasets/BZNSYP
```
Comment (zxcd) [inline examples/csmsc/voc1/README.md]: datasets/BZNSYP should have three folders:

└─ Wave
    └─ .wav files (audio speech)
└─ PhoneLabeling
    └─ .interval files (alignment between phoneme and duration)
└─ ProsodyLabeling
   └─ 000001-010000.txt (text with prosodic by pinyin)
Still we only use .wav files in training.

Comment (enkilee) [reply examples/csmsc/voc1/README.md]: ?我这边解压出来，只有wav文件和一个desktop.ini？

Comment (enkilee) [reply examples/csmsc/voc1/README.md]: 已按照赵总要求更改。

File: examples/csmsc/voc1/README.md [hunk 2]
```diff
@@ -3,6 +3,15 @@ This example contains code used to train a [parallel wavegan](http://arxiv.org/a
...
 Download CSMSC from it's [official website](https://test.data-baker.com/data/index/TNtts/) and extract it to `~/datasets`. Then the dataset is in the directory `~/datasets/BZNSYP`.
+datasets/BZNSYP should have three folders:
+
+└─ Wave
+  └─ .wav files (audio speech)
+  └─ PhoneLabeling
+      └─ .interval files (alignment between phoneme and duration)
+  └─ ProsodyLabeling
+    └─ 000001-010000.txt (text with prosodic by pinyin)
+Still we only use .wav files in training.
```
Comment (zxcd) [inline examples/csmsc/voc1/README.md]: Wave跟PhoneLabeling和ProsodyLabeling同级，以及口语化的表达修正一下

Comment (enkilee) [reply examples/csmsc/voc1/README.md]: roger

File: examples/csmsc/voc1/README.md [hunk 3]
```diff
@@ -4,6 +4,17 @@ This example contains code used to train a [parallel wavegan](http://arxiv.org/a
...
+After processing the data, the ``BZNSYP`` directory will look like this:
+```text
+BZNSYP
+├── Wave
+│    └─ *.wav files (audio speech)
+├── PhoneLabeling
+│    └─ *.interval files (alignment between phoneme and duration)
+└── ProsodyLabeling
+     └─ 000001-010000.txt (text with prosodic by pinyin)
+```
```
Comment (zxcd) [inline examples/csmsc/voc1/README.md:16]: Add `This experiment only uses *.wav files from the Wave file`

Comment (enkilee) [reply examples/csmsc/voc1/README.md:16]: roger

**Review by zxcd — APPROVED** [review]
LGTM

**Pull request merged**